user_guide_src/source/outgoing/response/018.php

```diff
@@ -1,4 +1,4 @@
 <?php
 
-$date = DateTime::createFromFormat('j-M-Y', '15-Feb-2016');
+$date = DateTime::createFromFormat('!j-M-Y', '15-Feb-2016');
 $response->setDate($date);
```

Comment:
What is the use case of this `Response::setDate()` method?
I feel that there is no need for it to be `public`.

Reply:
I could see it being helpful for functional or E2E testing where you want to "spoof" a past/future date. That's a bit of a reach though, bc I'm unclear how that would ever affect an outcome. 🤷‍♂️

Reply:
If you want to do E2E testing on a date in the past, you need a system clock (which provides the current time) and you must use it in all of your application (and framework).

Reply:
@paulbalandan @MGatner Why don't we just remove this sample code?
This method is not likely to be used (and it is easy to use if you want to use).

Reply:
Yeah, that sounds best.

Reply:
If this sample code would be removed, should the next samples be renumbered?

Reply:
We don't change the sample file names. 
If we want to renumber, we should do it to all files at some point.